sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: GPL-2.0
-// Copyright(c) 2019 Intel Corporation.
+// Copyright(c) 2019-2020 Intel Corporation.
 
 /*
  * Intel SOF Machine Driver with Realtek rt5682 Codec
```

Comment:
@yongzhi1 can you please also fix the comment on the next line to add that it supports RT1015 too?

Reply:
Ack, will update in next version.